### PR TITLE
show search box along with search results

### LINKED ISSUE #341: Website search results do not show original search query
When performing a search on the k8ssandra.io website, the search term is not repeated in the results. 

Screen Shot 2021-02-06 at 12 32 47 PM

Also the search results contain 3rd party ads, don't know if it is possible to remove these.

### PR DESCRIPTION
**Pull request by jeffreyscarpenter**
Fixes #341

### CODE REVIEW

File: docs/layouts/_default/header.html [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+                var s = document.getElementsByTagName('script')[0];
+                s.parentNode.insertBefore(gcse, s);
+            })();
+        </script>
+        <gcse:search></gcse:search>
+        {{ end }}
+    </div>
+</section>
+
+{{ end }}
```
Comment (bradfordcp) [inline docs/layouts/_default/header.html:22]: Missing newline at end of file

```suggestion
{{ end }}

```

**Review by bradfordcp — CHANGES_REQUESTED** [review]
What is the expected end state?

This is what I see when running locally.
image

**Comment by jeffreyscarpenter** [comment]
Should be fixed now, can you retest?

File: docs/layouts/_default/search.html [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+                var s = document.getElementsByTagName('script')[0];
+                s.parentNode.insertBefore(gcse, s);
+            })();
+        </script>
+        <gcse:search></gcse:search>
+        {{ end }}
+    </div>
+</section>
+
+{{ end }}
```
Comment (bradfordcp) [inline docs/layouts/_default/search.html:22]:
```suggestion
{{ end }}

```

**Review by bradfordcp — APPROVED** [review]

**Pull request merged**